# Single-Cell Model Collections
![](https://github.com/xianglin226/Benchmarking-Single-Cell-Perturbation/blob/master/mach.jpg)
Many models for single-cell perturbation data coming out!
## Models developed for single-cell perturbation data  
| Name   | Year          | Journal  | Title |
| -------|:-------------:| -----:|--------:|
| [Rachel et al](https://psb.stanford.edu/psb-online/proceedings/psb18/hodos.pdf)  |2018  |Pacific Symposium on Biocomputing 2018|Cell-specific prediction and application of drug-induced gene expression profiles  
| [scGEN](https://www.nature.com/articles/s41592-019-0494-8)|2019    |Nature Method|scGen predicts single-cell perturbation responses
| [DTD](https://dl.acm.org/doi/abs/10.1145/3308558.3313476) |2019         |The World Wide Web Conference, 2019|Modeling Relational Drug-Target-Disease Interactions via Tensor Factorization with Multiple Web Sources 
| [CPA](https://www.embopress.org/doi/full/10.15252/msb.202211517)|2021 |Molecular system biology|Predicting cellular responses to complex perturbations in high‚Äêthroughput screens 
| [CellBox](https://www.cell.com/cell-systems/pdf/S2405-4712(20)30464-6.pdf)|2021  |Cell systems|CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy 
| [CellDrift](https://academic.oup.com/bib/article/23/5/bbac324/6673850)|2022         |BIB | CellDrift: inferring perturbation responses in temporally sampled single-cell data
| [MultiCPA](https://www.biorxiv.org/content/10.1101/2022.07.08.499049v1.abstract)|2022          || MultiCPA: Multimodal Compositional Perturbation Autoencoder
| [PerturbNet](https://www.biorxiv.org/content/10.1101/2022.07.20.500854v2.abstract)|2022          || PerturbNet predicts single-cell responses to unseen chemical and genetic perturbations
| [scINSIGHT](https://link.springer.com/article/10.1186/s13059-022-02649-33)|2022        |Genome biology |scINSIGHT for interpreting single-cell gene expression from biologically heterogeneous data
| [scpregan](https://academic.oup.com/bioinformatics/article/38/13/3377/6593485)|2022    |Bioinformatics |scPreGAN, a deep generative model for predicting the response of single-cell expression to perturbation
| [Gears](https://www.nature.com/articles/s41587-023-01905-6)|2023          |Nature Biotech| Predicting transcriptional outcomes of novel multigene perturbations with GEARS
| [cycleCDR](https://arxiv.org/abs/2311.10315)|2023          || Interpretable Modeling of Single-cell perturbation Responses to Novel Drugs Using Cycle Consistence Learning
| [scVIDR](https://www.cell.com/patterns/pdf/S2666-3899(23)00186-1.pdf)|2023          |Patterns| Generative modeling of single-cell gene expression for dose-dependent chemical perturbations
| [Unagi](https://www.researchsquare.com/article/rs-3676579/v1)|2023           ||Unagi: Deep Generative Model for Deciphering Cellular Dynamics and In-Silico Drug Discovery in Complex Diseases
| [CINEMA-OT](https://www.nature.com/articles/s41592-023-02040-5)| 2023        |Nature Method| Causal identification of single-cell experimental perturbation effects with CINEMA-OT
| [ChemCPA](https://proceedings.neurips.cc/paper_files/paper/2022/hash/aa933b5abc1be30baece1d230ec575a7-Abstract-Conference.html)|2023        |NeurIPS 2022 |Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution 
| [DREEP](https://link.springer.com/article/10.1186/s12916-023-03182-1)| 2023        |BMC Medicine|Predicting drug response from single-cell expression profiles of tumours 
| [ontoVAE](https://academic.oup.com/bioinformatics/article/39/6/btad387/7199588)|2023         |Bioinformatics|  Biologically informed variational autoencoders allow predictive modeling of genetic and drug-induced perturbations
| [scDiff](https://www.biorxiv.org/content/10.1101/2023.10.13.562243v1.abstract)| 2023         || A GENERAL SINGLE-CELL ANALYSIS FRAMEWORK VIA CONDITIONAL DIFFUSION GENERATIVE MODELS
| [ContrastiveVI](https://www.nature.com/articles/s41592-023-01955-3)| 2023         |Nature Method|Isolating salient variations of interest in single-cell data with contrastiveVI 
| [sVAE](https://proceedings.mlr.press/v213/lopez23a.html)| 2023       |PMLR | Learning Causal Representations of Single Cells via Sparse Mechanism Shift Modeling
| [CellOT](https://www.nature.com/articles/s41592-023-01969-x)|2023         |Nature Method |Learning single-cell perturbation responses using neural optimal transport 
| [samsVAE](https://proceedings.neurips.cc/paper_files/paper/2023/hash/0001ca33ba34ce0351e4612b744b3936-Abstract-Conference.html)| 2024       |Advances in Neural Information Processing Systems| Modelling Cellular Perturbations with the Sparse Additive Mechanism Shift Variational Autoencoder 
| [STAMP](https://www.biorxiv.org/content/10.1101/2024.01.17.576034v1.abstract)|2024          ||Toward subtask decomposition-based learning and benchmarking for genetic perturbation outcome prediction and beyond 
| [Biolord](https://www.nature.com/articles/s41587-023-02079-x)| 2024        |Nature Biotech| Disentanglement of single-cell data with biolord
| [Pdgrapher](https://www.biorxiv.org/content/10.1101/2024.01.03.573985v2.abstract)|  2024       | | Combinatorial prediction of therapeutic perturbations using causally-inspired neural networks
| [TAT](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c01855)| 2024        |Journal of Chemical Information and Modeling| Compound Activity Prediction with Dose-Dependent Transcriptomic Profiles and Deep Learning 
| [scVAE](https://www.biorxiv.org/content/10.1101/2024.01.05.574421v1.abstract)|  2024      | | A Supervised Contrastive Framework for Learning Disentangled Representations of Cell Perturbation Data 
| [Cell PaintingCNN](https://www.nature.com/articles/s41467-024-45999-1)|2024        |NC |  Learning representations for image-based profiling of perturbations
| [scDisInFact](https://www.nature.com/articles/s41467-024-45227-w)| 2024       |NC| scDisInFact: disentangled learning for integration and prediction of multi-batch multi-condition single-cell RNA-sequencing data  
| [CellCap](https://www.biorxiv.org/content/10.1101/2024.03.14.585078v1.abstract)| 2024         || Modeling interpretable correspondence between cell state and perturbation response with CellCap
| [CODEX](https://academic.oup.com/bioinformatics/article/40/Supplement_1/i91/7700898) | 2024 | Bioinformatics | CODEX: COunterfactual Deep learning for the in silico EXploration of cancer cell line perturbations
  
# Perturbation Datasets  
[SC-perturb](http://projects.sanderlab.org/scperturb/)  
[C-MAP](https://clue.io/)  
  
## (Pretrained) (Large)(Language) Models developed for single-cell data  
| Name          | Year          | Journal  | Title |
| ------------- |:-------------:| -----:|--------:|
|[DeepMAPS](https://www.nature.com/articles/s41467-023-36559-0)|2021       | NC   | DeepMAPS: Single-cell biological network inference using heterogeneous graph transformer
|[scBERT](https://www.nature.com/articles/s42256-022-00534-z) | 2022      | NMI   | scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data
|[TransCluster](https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2022.1038919/full) | 2022      | Frontiers in Genetics   | TransCluster: A Cell-Type Identification Method for single-cell RNA-Seq data using deep learning based on transformer.
|[scMVP](https://link.springer.com/article/10.1186/s13059-021-02595-6)| 2022      | Genome Biology   | A deep generative model for multi-view profiling of single-cell RNA-seq and ATAC-seq data
|[scGPT](https://www.nature.com/articles/s41592-024-02201-0) |2023       | NM   | scGPT: Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI
|[Geneformer](https://www.nature.com/articles/s41586-023-06139-9) | 2023      | Nature   | Transfer learning enables predictions in network biology
|[CellLM](https://arxiv.org/pdf/2306.04371.pdf)  | 2023      |    | Large-Scale Cell Representation Learning via Divide-and-Conquer Contrastive Learning
|[Tgpt](https://www.cell.com/iscience/pdf/S2589-0042(23)00613-2.pdf)  | 2023     | Iscience   | Generative pretraining from large-scale transcriptomes for single-cell deciphering
|[Scimilarity](https://www.biorxiv.org/content/10.1101/2023.07.18.549537v3.abstract)  | 2023      |    | Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages
|[scFoundation](https://www.biorxiv.org/content/10.1101/2023.05.29.542705v4.abstract) | 2023      |    |  Large Scale Foundation Model on Single-cell Transcriptomics.
|[TOSICA](https://www.nature.com/articles/s41467-023-35923-4) | 2023      | NC   | Transformer for one stop interpretable cell type annotation
|[CIForm](https://academic.oup.com/bib/article/24/4/bbad195/7169137) | 2023      | BIB   | CIForm as a Transformer-based model for cell-type annotation of large-scale single-cell RNA-seq data
|[scTransSort](https://www.mdpi.com/2218-273X/13/4/611)  | 2023      | Biomolecules   | scTransSort: Transformers for Intelligent Annotation of Cell Types by Gene Embeddings
|[scMoFormer](https://dl.acm.org/doi/pdf/10.1145/3583780.3615061)  | 2023      |  ICIKM  | Single-Cell Multimodal Prediction via Transformers
|[scTranslator](https://www.biorxiv.org/content/10.1101/2023.07.04.547619v2.full.pdf)  | 2023      |    | A pre-trained large generative model for translating single-cell transcriptome to proteome
|[Cell2Sentence](https://www.biorxiv.org/content/biorxiv/early/2024/02/15/2023.09.11.557287.full.pdf)  | 2023      |    | Cell2Sentence: Teaching Large Language Models the Language of Biology
|[genePT](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10614824/pdf/nihpp-2023.10.16.562533v2.pdf)  |  2023     |    | GENEPT: A SIMPLE BUT HARD-TO-BEAT FOUNDATION MODEL FOR GENES AND CELLS BUILT FROM CHATGPT
|[scMulan](https://link.springer.com/chapter/10.1007/978-1-0716-3989-4_57)  |  2024     |ICRCMB| scMulan: a multitask generative pre-trained language model for single-cell analysis.

### These tables will be periodically updated. We will build APIs for some of these models on [TDC](https://tdcommons.ai/) for benchmarking.
### [CZI](https://cellxgene.cziscience.com/) single-cell database
