# Single-Cell Model Collections
![](https://github.com/xianglin226/Benchmarking-Single-Cell-Perturbation/blob/master/mach.jpg)
Many models for single-cell perturbation data coming out!
## Models developed for single-cell perturbation data  
| Name   | Year          | Journal  | Title |
| -------|:-------------:| -----:|--------:|
| [Rachel et al](https://psb.stanford.edu/psb-online/proceedings/psb18/hodos.pdf)  |2018  |Pacific Symposium on Biocomputing 2018|Cell-specific prediction and application of drug-induced gene expression profiles  
| [scGEN](https://www.nature.com/articles/s41592-019-0494-8)|2019    |Nature Method|scGen predicts single-cell perturbation responses
| [DTD](https://dl.acm.org/doi/abs/10.1145/3308558.3313476) |2019         |The World Wide Web Conference, 2019|Modeling Relational Drug-Target-Disease Interactions via Tensor Factorization with Multiple Web Sources 
| [CPA](https://www.embopress.org/doi/full/10.15252/msb.202211517)|2021 |Molecular system biology|Predicting cellular responses to complex perturbations in high‚Äêthroughput screens 
| [CellBox](https://www.cell.com/cell-systems/pdf/S2405-4712(20)30464-6.pdf)|2021  |Cell systems|CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy 
| [CellDrift](https://academic.oup.com/bib/article/23/5/bbac324/6673850)|2022         |BIB | CellDrift: inferring perturbation responses in temporally sampled single-cell data
| [MultiCPA](https://www.biorxiv.org/content/10.1101/2022.07.08.499049v1.abstract)|2022          || MultiCPA: Multimodal Compositional Perturbation Autoencoder
| [PerturbNet](https://www.biorxiv.org/content/10.1101/2022.07.20.500854v2.abstract)|2022          || PerturbNet predicts single-cell responses to unseen chemical and genetic perturbations
| [scINSIGHT](https://link.springer.com/article/10.1186/s13059-022-02649-33)|2022        |Genome biology |scINSIGHT for interpreting single-cell gene expression from biologically heterogeneous data
| [scpregan](https://academic.oup.com/bioinformatics/article/38/13/3377/6593485)|2022    |Bioinformatics |scPreGAN, a deep generative model for predicting the response of single-cell expression to perturbation
| [Gears](https://www.nature.com/articles/s41587-023-01905-6)|2023          |Nature Biotech| Predicting transcriptional outcomes of novel multigene perturbations with GEARS
| [cycleCDR](https://arxiv.org/abs/2311.10315)|2023          || Interpretable Modeling of Single-cell perturbation Responses to Novel Drugs Using Cycle Consistence Learning
| [scVIDR](https://www.cell.com/patterns/pdf/S2666-3899(23)00186-1.pdf)|2023          |Patterns| Generative modeling of single-cell gene expression for dose-dependent chemical perturbations
| [Unagi](https://www.researchsquare.com/article/rs-3676579/v1)|2023           ||Unagi: Deep Generative Model for Deciphering Cellular Dynamics and In-Silico Drug Discovery in Complex Diseases
| [CINEMA-OT](https://www.nature.com/articles/s41592-023-02040-5)| 2023        |Nature Method| Causal identification of single-cell experimental perturbation effects with CINEMA-OT
| [ChemCPA](https://proceedings.neurips.cc/paper_files/paper/2022/hash/aa933b5abc1be30baece1d230ec575a7-Abstract-Conference.html)|2023        |NeurIPS 2022 |Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution 
| [DREEP](https://link.springer.com/article/10.1186/s12916-023-03182-1)| 2023        |BMC Medicine|Predicting drug response from single-cell expression profiles of tumours 
| [ontoVAE](https://academic.oup.com/bioinformatics/article/39/6/btad387/7199588)|2023         |Bioinformatics|  Biologically informed variational autoencoders allow predictive modeling of genetic and drug-induced perturbations
| [scDiff](https://www.biorxiv.org/content/10.1101/2023.10.13.562243v1.abstract)| 2023         || A GENERAL SINGLE-CELL ANALYSIS FRAMEWORK VIA CONDITIONAL DIFFUSION GENERATIVE MODELS
| [ContrastiveVI](https://www.nature.com/articles/s41592-023-01955-3)| 2023         |Nature Method|Isolating salient variations of interest in single-cell data with contrastiveVI 
| [sVAE](https://proceedings.mlr.press/v213/lopez23a.html)| 2023       |PMLR | Learning Causal Representations of Single Cells via Sparse Mechanism Shift Modeling
| [CellOT](https://www.nature.com/articles/s41592-023-01969-x)|2023         |Nature Method |Learning single-cell perturbation responses using neural optimal transport 
|[MOASL](https://www.sciencedirect.com/science/article/abs/pii/S0010482523013185)| 2023| Computers in Biology and Medicine | MOASL: Predicting drug mechanism of actions through similarity learning with transcriptomic signature  
| [samsVAE](https://proceedings.neurips.cc/paper_files/paper/2023/hash/0001ca33ba34ce0351e4612b744b3936-Abstract-Conference.html)| 2024       |Advances in Neural Information Processing Systems| Modelling Cellular Perturbations with the Sparse Additive Mechanism Shift Variational Autoencoder 
| [Biolord](https://www.nature.com/articles/s41587-023-02079-x)| 2024        |Nature Biotech| Disentanglement of single-cell data with biolord
| [Pdgrapher](https://www.biorxiv.org/content/10.1101/2024.01.03.573985v2.abstract)|  2024       | | Combinatorial prediction of therapeutic perturbations using causally-inspired neural networks
| [TAT](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c01855)| 2024        |Journal of Chemical Information and Modeling| Compound Activity Prediction with Dose-Dependent Transcriptomic Profiles and Deep Learning 
| [scVAE](https://www.biorxiv.org/content/10.1101/2024.01.05.574421v1.abstract)|  2024      | | A Supervised Contrastive Framework for Learning Disentangled Representations of Cell Perturbation Data 
| [Cell PaintingCNN](https://www.nature.com/articles/s41467-024-45999-1)|2024        |NC |  Learning representations for image-based profiling of perturbations
| [scDisInFact](https://www.nature.com/articles/s41467-024-45227-w)| 2024       |NC| scDisInFact: disentangled learning for integration and prediction of multi-batch multi-condition single-cell RNA-sequencing data  
| [CellCap](https://www.biorxiv.org/content/10.1101/2024.03.14.585078v1.abstract)| 2024         || Modeling interpretable correspondence between cell state and perturbation response with CellCap
| [CODEX](https://academic.oup.com/bioinformatics/article/40/Supplement_1/i91/7700898) | 2024 | Bioinformatics | CODEX: COunterfactual Deep learning for the in silico EXploration of cancer cell line perturbations
| [scFM](https://www.biorxiv.org/content/10.1101/2024.10.02.616248v1) | 2024 |  | PertEval-scFM: Benchmarking Single-Cell Foundation Models for Perturbation Effect Prediction
| [STAMP](https://www.nature.com/articles/s43588-024-00698-1) | 2024 | NCS | Toward subtask-decomposition-based learning and benchmarking for predicting genetic perturbation outcomes and beyond  
| [PrePR-CT](https://www.biorxiv.org/content/10.1101/2024.07.24.604816v1.full.pdf) |2024| |  PrePR-CT: Predicting Perturbation Responses in Unseen Cell Types Using Cell-Type-Specific Graphs  
|[PRnet](https://www.nature.com/articles/s41467-024-53457-1) |2024| NC | Predicting transcriptional responses to novel chemical perturbations using deep generative model for drug discovery  
|[TranSiGen](https://www.nature.com/articles/s41467-024-49620-3)|2024|NC|Deep representation learning of chemical-induced transcriptional profile for phenotype-based drug discovery  
|[BioDiscoveryAgent](https://arxiv.org/abs/2405.17631)| 2024| | BioDiscoveryAgent: An AI Agent for Designing Genetic Perturbation Experiments  
|[DRSPRING](https://www.sciencedirect.com/science/article/pii/S0010482524005201)| 2024| Computers in Biology and Medicine| DRSPRING: Graph convolutional network (GCN)-Based drug synergy prediction utilizing drug-induced gene expression profile  
|[PertKGE](https://www.biorxiv.org/content/10.1101/2024.04.08.588632v1.abstract)|2024|| Identify compound-protein interaction with knowledge graph embedding of perturbation transcriptomics  
|[scRank](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(24)00260-X)| 2024| Cell Reports Medicine | scRank infers drug-responsive cell types from untreated scRNA-seq data using a target-perturbed gene regulatory network  
|[Pertpy](https://www.biorxiv.org/content/10.1101/2024.08.04.606516v1.full.pdf)| 2024 | | Pertpy: an end-to-end framework for perturbation analysis  
|[CellFlow](https://www.biorxiv.org/content/10.1101/2025.04.11.648220v1)| 2025 | | CellFlow enables generative single-cell phenotype modeling with flow matching  
|[TxPert](https://arxiv.org/abs/2505.14919)| 2025 | | TxPert: Leveraging Biochemical Relationships for Out-of-Distribution Transcriptomic Perturbation Prediction  
|[IMPA](https://www.nature.com/articles/s41467-024-55707-8) |2025| NC | Predicting cell morphological responses to perturbations using generative modeling  
|[PS](https://www.nature.com/articles/s41556-025-01626-9) |2025| Nature Cell Biology | Decoding heterogeneous single-cell perturbation responses  
|[TRADE](https://www.nature.com/articles/s41588-025-02169-3) |2025| Nature Genetics | Transcriptome-wide analysis of differential expression in perturbation atlases    
|[STATE](https://arcinstitute.org/manuscripts/State) |2025|  | Predicting cellular responses to perturbation across diverse contexts with State   
|[scAgent](https://openreview.net/forum?id=HGJQvwGtfJ) |2025|  | scAgents:AMulti-AgentFramework forFullyAutonomousEnd-to-EndSingle-Cell PerturbationAnalysis  
  
## Perturbation Datasets  
[SC-perturb](http://projects.sanderlab.org/scperturb/)  
[C-MAP](https://clue.io/)  
[PerturbBase](http://www.perturbase.cn/)  
[PerturDB](http://research.gzsys.org.cn/perturbdb/#/home)  
[Tahoe-100M](https://huggingface.co/datasets/tahoebio/Tahoe-100M)  
[Multiome Perturb-seq](https://www.biorxiv.org/content/10.1101/2024.07.26.605307v1.full.pdf) (paper)  
[Multiome: CRISPRmap](https://www.nature.com/articles/s41587-024-02386-x) (paper)  
[Spatial: Perturb-Fish](https://www.cell.com/cell/fulltext/S0092-8674(25)00197-7) (paper)  
[Spatial: PerturbView](https://www.nature.com/articles/s41587-024-02391-0) (paper)  
[Spatial: Perturb-map](https://www.cell.com/cell/fulltext/S0092-8674(22)00195-7?) (paper)   
[Spatial: Perturb-DBiT](https://www.biorxiv.org/content/10.1101/2024.11.18.624106v1.abstract) (paper)  
[Spatial: Spatial-perturb-seq](https://www.biorxiv.org/content/biorxiv/early/2024/12/20/2024.12.19.628843.full.pdf) (paper)  
[Spatial: NIS-seq](https://www.nature.com/articles/s41587-024-02516-5.pdf) (paper)  

### List of [scFM](https://github.com/xianglin226/Benchmarking-Single-Cell-Perturbation/blob/master/scFM/README.md)  
### These tables will be periodically updated. 
### We will build APIs for some of these models on [TDC](https://tdcommons.ai/) for benchmarking. 
### Find single-cell data from [CZI](https://cellxgene.cziscience.com/) database

## FastPert  
A tool to quickly check:  
- DEGs of perturbations: [FastPert](https://benchmarking-single-cell-perturbation-cxpnztwdkmkuhrqv555ut8.streamlit.app/)  
- Enriched pathways of perturbations: [FastPert](https://benchmarking-single-cell-perturbation-h6zgs5utz5cbk9ckdavgxf.streamlit.app/)  

## Questions, ideas, comments, ...
Please contact Xiang Lin (xiang_lin@hms.harvard.edu).

## Citation is appeciated
@misc{pertogether2025,  
  title  = {PerTogether: A tool for perturbation research and modeling},  
  author = {Xiang Lin},  
  year   = {2025},  
  url    = {https://github.com/xianglin226/Benchmarking-Single-Cell-Perturbation/},  
  note   = {Version 1.0}  
}
