## (Pretrained) (Large)(Language) Models developed for single-cell data  
| Name          | Year          | Journal  | Title |
| ------------- |:-------------:| -----:|--------:|
|[DeepMAPS](https://www.nature.com/articles/s41467-023-36559-0)|2021       | NC   | DeepMAPS: Single-cell biological network inference using heterogeneous graph transformer
|[scBERT](https://www.nature.com/articles/s42256-022-00534-z) | 2022      | NMI   | scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data
|[TransCluster](https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2022.1038919/full) | 2022      | Frontiers in Genetics   | TransCluster: A Cell-Type Identification Method for single-cell RNA-Seq data using deep learning based on transformer.
|[scMVP](https://link.springer.com/article/10.1186/s13059-021-02595-6)| 2022      | Genome Biology   | A deep generative model for multi-view profiling of single-cell RNA-seq and ATAC-seq data
|[scGPT](https://www.nature.com/articles/s41592-024-02201-0) |2023       | NM   | scGPT: Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI
|[Geneformer](https://www.nature.com/articles/s41586-023-06139-9) | 2023      | Nature   | Transfer learning enables predictions in network biology
|[CellLM](https://arxiv.org/pdf/2306.04371.pdf)  | 2023      |    | Large-Scale Cell Representation Learning via Divide-and-Conquer Contrastive Learning
|[Tgpt](https://www.cell.com/iscience/pdf/S2589-0042(23)00613-2.pdf)  | 2023     | Iscience   | Generative pretraining from large-scale transcriptomes for single-cell deciphering
|[Scimilarity](https://www.biorxiv.org/content/10.1101/2023.07.18.549537v3.abstract)  | 2023      |    | Scalable querying of human cell atlases via a foundational model reveals commonalities across fibrosis-associated macrophages
|[scFoundation](https://www.biorxiv.org/content/10.1101/2023.05.29.542705v4.abstract) | 2023      |    |  Large Scale Foundation Model on Single-cell Transcriptomics.
|[TOSICA](https://www.nature.com/articles/s41467-023-35923-4) | 2023      | NC   | Transformer for one stop interpretable cell type annotation
|[CIForm](https://academic.oup.com/bib/article/24/4/bbad195/7169137) | 2023      | BIB   | CIForm as a Transformer-based model for cell-type annotation of large-scale single-cell RNA-seq data
|[scTransSort](https://www.mdpi.com/2218-273X/13/4/611)  | 2023      | Biomolecules   | scTransSort: Transformers for Intelligent Annotation of Cell Types by Gene Embeddings
|[scMoFormer](https://dl.acm.org/doi/pdf/10.1145/3583780.3615061)  | 2023      |  ICIKM  | Single-Cell Multimodal Prediction via Transformers
|[scTranslator](https://www.biorxiv.org/content/10.1101/2023.07.04.547619v2.full.pdf)  | 2023      |    | A pre-trained large generative model for translating single-cell transcriptome to proteome
|[Cell2Sentence](https://www.biorxiv.org/content/biorxiv/early/2024/02/15/2023.09.11.557287.full.pdf)  | 2023      |    | Cell2Sentence: Teaching Large Language Models the Language of Biology
|[genePT](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10614824/pdf/nihpp-2023.10.16.562533v2.pdf)  |  2023     |    | GENEPT: A SIMPLE BUT HARD-TO-BEAT FOUNDATION MODEL FOR GENES AND CELLS BUILT FROM CHATGPT
|[scMulan](https://link.springer.com/chapter/10.1007/978-1-0716-3989-4_57)  |  2024     |ICRCMB| scMulan: a multitask generative pre-trained language model for single-cell analysis.
|[Nicheformer](https://www.biorxiv.org/content/10.1101/2024.04.15.589472v1)  |  2024     | | Nicheformer: a foundation model for single-cell and spatial omics.  
|[CellPLM](https://openreview.net/forum?id=BKXvPDekud)  |  2024     | ICLR | CellPLM: Pre-training of Cell Language Model Beyond Single Cells  
